distribution/docs/src/main/resources/content/_reference/_tables/org.codice.ddf.transformer.preview.adoc

```diff
@@ -0,0 +1,33 @@
+:title: Catalog Preview
+:id: org.codice.ddf.transformer.preview
+:type: table
+:status: published
+:application: ${ddf-catalog}
+:summary: Allow Preview to be Extracted From Metadata.
+
+.[[_org.codice.ddf.transformer.preview]]Catalog Preview
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+
+|Preview From Metadata
+|previewFromMetadata
+|Boolean
+|Select whether metacard's previews should be searched when creating a preview in Inspector
```

Comment:
Make this consistent with the metatype description `Allow Preview to be Extracted From Metadata.`